dev-util/sourcetrail/sourcetrail-2019.4.102.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A free and open-source cross-platform source explorer"
+DESCRIPTION="Sourcetrail helps you get productive on unfamiliar source code"
+HOMEPAGE="https://www.sourcetrail.com/"
+COPYRIGHT="2019 Andreas Stallinger and others"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/CoatiSoftware/Sourcetrail/archive/2019.4.102.tar.gz"
+CHECKSUM_SHA256="74b0f11904e5faaead94f92319001d77e23d102c7370f63f0f83a624e46312fe"
+SOURCE_DIR=""
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+ 	sourcetrail$secondaryArchSuffix = $portVersion
+ 	cmd:sourcetrail
+ 	app:sourcetrail
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5$secondaryArchSuffix >= 5.12
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Designer$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5$secondaryArchSuffix >= 5.12
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Designer$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libboost_filesystem$secondaryArchSuffix >= 1.67
+	devel:libboost_iostreams$secondaryArchSuffix >= 1.67
+	devel:libboost_program_options$secondaryArchSuffix >= 1.67
+	devel:libboost_system$secondaryArchSuffix >= 1.67
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:getconf
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p $prefix
+	cd $prefix
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_BUILD_TYPE="Release" -DBOOST_ROOT=$portPackageLinksDir/devel~libboost_program_options/develop -DQt5_DIR=$portPackageLinksDir/devel~Qt/version/platform/compiler/lib/cmake/Qt5 $sourceDir
+}
+
+INSTALL()
+{
+	make Sourcetrail PREFIX=$prefix
```

Comment:
What does this PREFIX does? How will make find its makefile since you generated it in another directory using cmake?